### PR TITLE
Enable Networking on OP/Base archive nodes

### PR DESCRIPTION
**Pull request by kamilchodola**
Networking was disabled on archive nodes which is invalid.

#### What types of changes does your code introduce?

- [X] Bugfix (a non-breaking change that fixes an issue)
- [ ] New feature (a non-breaking change that adds functionality)
- [ ] Breaking change (a change that causes existing functionality not to work as expected)
- [ ] Optimization
- [ ] Refactoring
- [ ] Documentation update
- [ ] Build-related changes
- [ ] Other: _Description_

#### Requires testing

- [X] Yes
- [ ] No

#### If yes, did you write tests?

- [ ] Yes
- [X] No

### CODE REVIEW

**Review by LukaszRozmej — APPROVED** [review]

**Comment by emlautarom1** [comment]
Relevant: https://github.com/NethermindEth/nethermind/pull/6334

Could you elaborate on what changed that now we're enabling networking?

File: src/Nethermind/Nethermind.Runner/configs/op-mainnet_archive.json [hunk 1]
```diff
@@ -10,11 +10,6 @@
   "TxPool": {
     "BlobsSupport": "Disabled"
   },
-  "Sync": {
-    "NetworkingEnabled": false,
-    "AncientBodiesBarrier": 105235063,
```
Comment (MarekM25) [inline src/Nethermind/Nethermind.Runner/configs/op-mainnet_archive.json:15]: Barrier not needed anymore?

**Review by MarekM25 — APPROVED** [review]

File: src/Nethermind/Nethermind.Runner/configs/op-mainnet_archive.json (hunk 1 above)
Comment (kamilchodola) [reply src/Nethermind/Nethermind.Runner/configs/op-mainnet_archive.json:15]: No - especially on archive config xD Not sure why it was even there

Comment (kamilchodola) [reply src/Nethermind/Nethermind.Runner/configs/op-mainnet_archive.json:15]: Actually learnt today about Regenesis of op-mainnet.
So reverted barriers

**Review by jmederosalvarado — APPROVED** [review]
Enabling Networking - Looks good

**Pull request merged**